karate-core/src/main/java/com/intuit/karate/XmlUtils.java

```diff
@@ -111,6 +111,7 @@ public static Document toXmlDoc(String xml) {
     public static Document toXmlDoc(String xml, boolean namespaceAware) {
         DocumentBuilderFactory factory = DocumentBuilderFactory.newInstance();
         factory.setNamespaceAware(namespaceAware);
+        factory.setIgnoringElementContentWhitespace(false);
```

Comment:
`false` is  the default setting. So nothing changes in the current document parsing  behavior. 
Added the above line to make the behavior explicit.